samples/callbacks/Callbacks_3/WebSender/HomeController.cs

```diff
@@ -17,8 +24,7 @@ public async Task<ActionResult> SendEnumMessage()
         var message = new EnumMessage();
         var sendOptions = new SendOptions();
         sendOptions.SetDestination("Samples.Callbacks.Receiver");
-        var endpointInstance = MvcApplication.EndpointInstance;
-        var statusTask = endpointInstance.Request<Status>(message, sendOptions);
+        var statusTask = messageSession.Request<Status>(message, sendOptions);
```

Comment:
I would find 

```
var status = await messageSession.Request<Status>(message, sendOptions);
```

more user-friendly for a sample. Comment applies for all the other cases too.

Reply:
Good call. I'll update these.